compositor_render/src/gpu_shader.rs

```diff
@@ -121,7 +136,7 @@ impl GpuShader {
     ) {
         let ctx = &self.wgpu_ctx;
 
-        // TODO: sources need to be ordered
+        // TODO: sources/textures need to be ordered
```

Comment:
Is this TODO still needed? Aren't sources already ordered?
cc: @WojciechBarczynski